### PR TITLE
GDSC-108 Email and address cards are working

### PR DESCRIPTION
**Pull request by MoNNsterFTW**

### CODE REVIEW

**Comment by MoNNsterFTW** [comment]
Screenshot_423
Screenshot_424
Screenshot_425
Screenshot_426
Screenshot_427
Screenshot_428
Screenshot_429

File: src/app/pages/contact-page/contact-page.component.scss [hunk 1]
```diff
@@ -0,0 +1,111 @@
+.contact-info-area {
+  background-color: #edf5ff;
+  padding: 20px;
+}
+
+.container {
+  width: 100%;
+  margin-right: auto;
+  margin-left: auto;
```
Comment (edi334) [inline src/app/pages/contact-page/contact-page.component.scss:9]: use margin: 0 auto instead

File: src/app/pages/contact-page/contact-page.component.scss [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+}
+
+.row {
+  flex-wrap: wrap;
+  margin-right: -30%;
+  margin-left: 10%;
```
Comment (edi334) [inline src/app/pages/contact-page/contact-page.component.scss:15]: use margin: 0 -30% 0 10% instead

File: src/app/pages/contact-page/contact-page.component.scss [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+}
+
+.contact-info-box{
+  margin-left: 1%;
+  box-shadow: 0 2px 48px 0 rgb(0 0 0 / 8%);
+  background-color: white;
+  border: 0;
+  padding: 2% 10% 2% 10%;
+  margin-bottom: 30px;
```
Comment (edi334) [inline src/app/pages/contact-page/contact-page.component.scss:24]: use margin: 0 0 30px 1%; instead

File: src/app/pages/contact-page/contact-page.component.scss [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    flex-shrink: 0;
+    flex-basis: 50%;
+    max-width: 50%;
+  }
+}
+
+@media only screen and (max-width: 767px) {
+  .contact-info-area {
+    padding-bottom: 40px;
+    padding-left: 0;
```
Comment (edi334) [inline src/app/pages/contact-page/contact-page.component.scss:93]: use padding: 0 0 40px 0; instead

File: src/app/pages/contact-page/contact-page.component.scss [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+  }
+  .contact-info-box {
+    margin-left: 0;
+    margin-right: 2%;
+    margin-bottom: 30px;
```
Comment (edi334) [inline src/app/pages/contact-page/contact-page.component.scss:98]: use margin: 0 2% 30px 0; instead

**Review by edi334 — CHANGES_REQUESTED** [review]
Please center the button. Also on some pixel size (455px) for example it looks weird so make sure to cover all the cases
Capture

**Review by edi334 — APPROVED** [review]

**Pull request merged**